Library/Homebrew/tap.rb

```diff
@@ -1102,6 +1102,12 @@ def formula_file_to_name(file)
     file.basename(".rb").to_s
   end
 
+  # @private
+  sig { params(file: Pathname).returns(String) }
+  def cask_file_to_name(file)
+    file.basename(".rb").to_s
+  end
```

Comment:
If it's `@private` probably shouldn't use in `brew test-bot`. The method is simple enough a version could just live in there.

Similarly, would be nice to combine these two functions if we need to have two rather than them having an identical body.